src/main/scala/edu/ie3/osmogrid/guardian/run/RunSupport.scala

```diff
@@ -6,21 +6,34 @@
 
 package edu.ie3.osmogrid.guardian.run
 
-import org.apache.pekko.actor.typed.ActorRef
-import org.apache.pekko.actor.typed.scaladsl.ActorContext
-import edu.ie3.osmogrid.cfg.OsmoGridConfig.{Generation, Output}
+import edu.ie3.osmogrid.cfg.OsmoGridConfig.Generation.Lv.{
+  BoundaryAdminLevel,
+  Osm
+}
+import edu.ie3.osmogrid.cfg.OsmoGridConfig.Generation.{Lv, Mv}
+import edu.ie3.osmogrid.cfg.OsmoGridConfig.Output
 import edu.ie3.osmogrid.cfg.{ConfigFailFast, OsmoGridConfig}
-import edu.ie3.osmogrid.exception.UnsupportedRequestException
 import edu.ie3.osmogrid.io.input.{InputDataEvent, InputDataProvider}
 import edu.ie3.osmogrid.io.output.{ResultListener, ResultListenerProtocol}
 import edu.ie3.osmogrid.lv.{LvCoordinator, LvRequest, LvResponse, ReqLvGrids}
 import edu.ie3.osmogrid.mv.{MvCoordinator, MvRequest, MvResponse, ReqMvGrids}
+import org.apache.pekko.actor.typed.ActorRef
+import org.apache.pekko.actor.typed.scaladsl.ActorContext
 
 import java.util.UUID
-import scala.util.{Failure, Success, Try}
+import scala.util.{Success, Try}
 
 trait RunSupport {
 
+  private val lvFallback: Lv = Lv(
+    averagePowerDensity = 2000,
```

Comment:
2000 W/m² is quite a lot :) 
```suggestion
    averagePowerDensity = 2,
```

Reply:
The OSM data may contain houses that are given as a node. For those we need this high average power density of 2 kW. If we change this value, there is only one lv subgrid for `fuerweiler`.

Reply:
Ok, we should handle this case as well. But this can be done in a new Issue. Could you please create one. Thanks!

Reply:
I will create an issue for this.